airship/Rotor left
airship/Rotor right
